win32ss/user/conime/conime.rc

```diff
@@ -0,0 +1,39 @@
+#include <windef.h>
+#include <winuser.h>
+#include "resource.h"
+
+LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
+
+#define REACTOS_STR_FILE_DESCRIPTION  "ReactOS Console IME"
+#define REACTOS_STR_INTERNAL_NAME     "conime"
+#define REACTOS_STR_ORIGINAL_FILENAME "conime.exe"
+#define REACTOS_FILEVERSION           5, 2, 3790, 3959
+#define REACTOS_PRODUCTVERSION        5, 2, 3790, 3959
```

Comment:
Why do you need specifying version numbers?

Reply:
Deleted.